Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py

```diff
@@ -13,7 +1038,17 @@ def return_json_entry(obj):
 
 
 def main():
-    return_json_entry({})
+    incidents_query_args = demisto.args()
```

Comment:
default modified > last run time
first run - run on last 6 months with max of 1,000-3,000
we can store the state in list: state will store last run time